client/boot/index.js

```diff
@@ -163,6 +162,7 @@ function boot() {
 		// Create layout instance with current user prop
 		Layout = require( 'layout' );
 		layout = ReactDom.render( React.createElement( Layout, {
+			store: reduxStore,
```

Comment:
I dont know if we really need that - we can connect components that need access to store using connect